vue.config.js

```diff
@@ -1,5 +1,24 @@
+const CompressionPlugin = require("compression-webpack-plugin");
+
 module.exports = {
   devServer: {
-    proxy: process.env.BASE_URL
+    proxy: process.env.BASE_URL,
+    port: 8000
+  },
+  productionSourceMap: false,
+  configureWebpack: config => {
+    if (process.env.NODE_ENV === "production") {
+      config.plugins.push(
+        new CompressionPlugin({
+          deleteOriginalAssets: true
```

Comment:
Just curious, why is this needed?

Reply:
This is the gzip compression and we only want that for the build script. We dont need the sourcemaps for  production and they add to our footprint. We also dont need thr non-compressed files, so deleting those from  dist.